src/cli.ts

```diff
@@ -1,13 +1,27 @@
 #!/usr/bin/env node
 
 import {whiteBright} from 'cli-color'
-import {JSONSchema4} from 'json-schema'
 import minimist = require('minimist')
-import {readFile, writeFile} from 'mz/fs'
-import {resolve} from 'path'
+import {readFile, writeFile, existsSync, lstatSync, readdirSync} from 'mz/fs'
+import * as _mkdirp from 'mkdirp'
+import * as _glob from 'glob'
+import isGlob = require('is-glob')
+import {promisify} from 'util'
+import {join, resolve, dirname, basename} from 'path'
 import stdin = require('stdin')
 import {compile, Options} from './index'
 
+// Promisify mkdirp & glob
+const mkdirp = (path: string) =>
```

Comment:
I would consider using `fsPromises.mkdir(path, {recursive: true})` instead - see https://nodejs.org/api/fs.html#fs_fspromises_mkdir_path_options

Reply:
if I remember correctly that option only exists on node version 10+ but this lib needs to support node 8 so I wasn't able to use it